pkgs/applications/misc/activate-linux/default.nix

```diff
@@ -24,19 +27,38 @@ stdenv.mkDerivation rec {
   ];
```

Comment:
```suggestion
  wayland-scanner
  ];
```

wayland-scanner is used

Reply:
Fixed

Reply:
it's not in the dependencies in the README but it's in the source code I guess.